### PR TITLE
ADD: added result page

### PR DESCRIPTION
**Pull request by zoef-yt**
In this pr, I added

Result page.

### CODE REVIEW

File: pages/questionpage/questionpage.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	color: var(--colour-black-second);
+}
+.answers .answer-correct:hover {
+	background-color: var(--colour-accent-one);
+}
+
+.answers .answer-wrong {
+	background-color: var(--colour-error);
+	color: var(--colour-black-second);
+}
```
Comment (bhushannaikaroskar) [inline pages/questionpage/questionpage.css:42]: Hey zoef you can name either answer-correct as answer-right or answer-wrong as .answer-incorrect. Like the naming will look more like  perfect opposite work.

File: pages/questionpage/questionpage.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	margin: 2rem 12rem;
+}
+.question-section div {
+	margin: 1rem 0;
+}
+
+.question-section div .btn-outline-secondary:hover {
+	background-color: transparent;
+	cursor: default;
+	color: var(--colour-primary);
```
Comment (bhushannaikaroskar) [inline pages/questionpage/questionpage.css:11]: Maybe you can avoid this cursor: default. As it is set to default anyways.

**Review by bhushannaikaroskar — CHANGES_REQUESTED** [review]
Hey zoef reviewed your code. I have suggested some changes please do check.

**Review by iRohitGaur — APPROVED** [review]
Clean code. Good to merge 👍🏻

**Review by sumitk-singh — APPROVED** [review]
-Good use of class name.
-Clean and readable code.
Good to go.

File: pages/questionpage/questionpage.css (hunk 2 above)
Comment (zoef-yt) [reply pages/questionpage/questionpage.css:11]: This is to override the default.

File: pages/questionpage/questionpage.css (hunk 1 above)
Comment (zoef-yt) [reply pages/questionpage/questionpage.css:42]: done.

**Pull request merged**